### PR TITLE
Fix epub doc configuration

### PR DESCRIPTION
**Pull request by andreas-bok-sociomantic**
Some characters were written as html entity names.
Those characters are &nbsp; and &copy have been
encoded with their equivalent html entity number

Files not relevant for the epub build have been
excluded as well using the 'epub_exclude_files'
setting.

### CODE REVIEW

**Comment by ben-palmer-sociomantic** [comment]
I would change the first paragraph of the commit message to:
```
Some characters were written as html entity names 
(&nbsp; and &copy). They are now encoded with their 
equivalent html entity number
```

File: docs/conf.py.in [hunk 1]
```diff
@@ -222,6 +222,8 @@ latex_elements = {
     'classoptions': 'openany,oneside', 'babel' : '\\usepackage[english]{babel}'
 }
 
+epub_exclude_files = ['.doctrees/compiling.doctree', '.doctrees/examples/statement.doctree']
```
Comment (ben-palmer-sociomantic) [inline docs/conf.py.in:225]: I would split these changes in to a separate commit.

**Comment by andreas-bok-sociomantic** [comment]
- Updated ci message 
- Split into two commits 
- Removed the `html_use_smartypants = True` change since it is `True` by default

**Comment by ben-palmer-sociomantic** [comment]
The first commit message could still be updated then LGTM.

**Comment by andreas-bok-sociomantic** [comment]
> The first commit message could still be updated then LGTM.

True, lost in rebase :)

**Comment by andreas-bok-sociomantic** [comment]
ping
